docs/guides/error-mitigation-and-suppression-techniques.ipynb

```diff
@@ -278,6 +278,23 @@
     "estimator.options.resilience.pec.max_overhead = 100"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "a6c87cd7",
+   "metadata": {},
+   "source": [
+    "## When to use each method\n",
+    "\n",
+    "| Method | When to use | Notes|\n",
+    "| --- | --- | --- |\n",
+    "| Dynamical decoupling | Use with circuits in which one or more qubits have long idle times | Circuits with dense structure of two-qubit gates (for example, see the paper [Evidence for the utility of quantum computing before fault tolerance](https://doi.org/10.1038/s41586-023-06096-3)). Also, Heron devices are less susceptible to the dominating noise that is suppressed by simple DD sequences (cross-talk) |\n",
+    "| Pauli twirling | Always use | There is a trade-off in the execution time (devices execute more shots faster than more circuits); therefore, using Pauli twirling with enough randomization (even when lowering the shots to keep the total number of shots constant) will result in a slowdown |\n",
+    "| TREX | Always use, if already Pauli twirling the circuits | There is some indication that TREX can lead to unphysical expectation values in some cases, but the risk is likely worth it, given the strength of measurement noise and low overhead |\n",
+    "| ZNE | Use if the noisy expectation value measured without ZNE is at least 30% of the ideal value (otherwise, extrapolation is likely to break down, leading to biased results)| Increases runtime by X times (where X is the number of noise factors) |\n",
+    "| PEA | Use if the noisy expectation value measured without ZNE is at least 30% of the ideal value (otherwise extrapolation is likely to break down, leading to biased results); and if the number of unique layers in the circuit is relatively small (fewer than 20)| Increases runtime by X times (where X is the number of noise factors), and needs time for learning the noise model |\n",
```

Comment:
```suggestion
    "| PEA | Use if the noisy expectation value measured without ZNE is at least 30% of the ideal value (otherwise extrapolation is likely to break down, leading to biased results); and if there are fewer than 20 unique layers in the circuit.| Increases runtime by X times, where X is the number of noise factors, and needs time to learn the noise model |\n",
```